### ISSUE TITLE
[BUG] text search consistently fail due to 202 error, all other methods work

### ISSUE THREAD

**Issue by sorendaugaard** [deedy5/duckduckgo_search#77]
**Reinstall duckduckgo_search and сheck if the problem is gone**

```pip install -I duckduckgo_search```

```pip install -I duckduckgo_search --break-system-packages```

**Describe the bug**

**To Reproduce**
```
from duckduckgo_search import DDGS
ddgs = DDGS()
answer = ddgs.answers("Colin Hanks")
a = answer.__next__()
print(a)
> {'icon': None, 'text': 'Colin Lewes Hanks is an American actor. He has starred in films including Orange County, King Kong, The House Bunny, The Great Buck Howard, and the Jumanji film series. His television credits include Roswell, Band of Brothers, Dexter, Fargo, The Good Guys, Life in Pieces, Impeachment: American Crime Story, The Offer, and A Friend of the Family. He is the eldest son of actor Tom Hanks.', 'topic': None, 'url': 'https://en.wikipedia.org/wiki/Colin_Hanks'}

results = ddgs.text("Colin Hanks")
r = results.__next__()

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "....-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 146, in text
    resp = self._get_url(
           ^^^^^^^^^^^^^^
  File "....-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 69, in _get_url
    raise ex
  File "....-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 62, in _get_url
    raise requests.HTTPError
requests.exceptions.HTTPError
```

If I debug, It seems to be because the response code in `duckduckgo_search.py` on line 61 is 202. 

**Expected behavior**
Should return a result. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - duckduckgo_search version 3.3.0

**Additional context**
Run this code and show results, change text() to the function that is causing the problem
```python
import logging
from duckduckgo_search import DDGS

logging.basicConfig(level=logging.DEBUG)

for r in DDGS().text("something"):
    pass
```

```
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): duckduckgo.com:443
DEBUG:urllib3.connectionpool:https://duckduckgo.com:443 "POST / HTTP/1.1" 200 None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): links.duckduckgo.com:443
DEBUG:urllib3.connectionpool:https://links.duckduckgo.com:443 "GET /d.js?q=something&kl=wt-wt&l=wt-wt&s=0&vqd=4-174670497610453878004351319754753261514&o=json&p=&sp=0&ex=-1 HTTP/1.1" 202 None
WARNING:duckduckgo_search.duckduckgo_search:_get_url() https://links.duckduckgo.com/d.js HTTPError
DEBUG:urllib3.connectionpool:https://links.duckduckgo.com:443 "GET /d.js?q=something&kl=wt-wt&l=wt-wt&s=0&vqd=4-174670497610453878004351319754753261514&o=json&p=&sp=0&ex=-1 HTTP/1.1" 202 None
WARNING:duckduckgo_search.duckduckgo_search:_get_url() https://links.duckduckgo.com/d.js HTTPError
DEBUG:urllib3.connectionpool:https://links.duckduckgo.com:443 "GET /d.js?q=something&kl=wt-wt&l=wt-wt&s=0&vqd=4-174670497610453878004351319754753261514&o=json&p=&sp=0&ex=-1 HTTP/1.1" 202 None
WARNING:duckduckgo_search.duckduckgo_search:_get_url() https://links.duckduckgo.com/d.js HTTPError
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/soren.daugaard/Library/Caches/pypoetry/virtualenvs/operation-prospect-peeper-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 146, in text
    resp = self._get_url(
           ^^^^^^^^^^^^^^
  File "/Users/soren.daugaard/Library/Caches/pypoetry/virtualenvs/operation-prospect-peeper-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 69, in _get_url
    raise ex
  File "/Users/soren.daugaard/Library/Caches/pypoetry/virtualenvs/operation-prospect-peeper-iFP_gLpf-py3.11/lib/python3.11/site-packages/duckduckgo_search/duckduckgo_search.py", line 62, in _get_url
    raise requests.HTTPError
requests.exceptions.HTTPError
```

**Comment by deedy5** [comment]
Try using a proxy, the site started randomly block ip according to an unknown algorithm.

**Comment by deedy5** [comment]
I issued a fix, update to version v3.4.0
